core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/source/scan/json/JSONUtil.scala

```diff
@@ -0,0 +1,34 @@
+package edu.uci.ics.texera.workflow.operators.source.scan.json
+import com.fasterxml.jackson.databind.JsonNode
+
+import scala.jdk.CollectionConverters.asScalaIteratorConverter;
+
+object JSONUtil {
+  @throws[RuntimeException]
+  def parseJSON(
+      node: JsonNode,
+      flatten: Boolean,
+      parentName: String = ""
+  ): Map[String, String] = {
+    var result = Map[String, String]()
+    if (node.isObject)
```

Comment:
Pls use bracket after if. It helps with code readability

Reply:
done